mlflow/spark/__init__.py

```diff
@@ -1149,7 +1149,6 @@ def predict(
         return [x.prediction for x in prediction_df.collect()]
 
 
-@track_api_usage
```

Comment:
I'm removing this to unblock this PR for now, there's a strange error when applying the telemetry tracking on spark autologging (https://github.com/mlflow/mlflow/actions/runs/16286878493/job/45987463978?pr=16439#step:13:305)